evaluator/localization_evaluator/launch/localization_evaluator.launch.xml

```diff
@@ -1,9 +1,10 @@
 <launch>
   <arg name="input/localization" default="/localization/kinematic_state"/>
   <arg name="input/localization/ref" default="/geometry_msgs/PoseWithCovarianceStampedGt"/>
+  <param name="param_file" value="$(find-pkg-share localization_evaluator)/config/localization_evaluator.param.yaml"/>
 
   <node name="localization_evaluator" exec="localization_evaluator" pkg="localization_evaluator" output="screen">
-    <param from="$(find-pkg-share localization_evaluator)/param/localization_evaluator.defaults.yaml"/>
```

Comment:
Are there any reasons for making this change?
I think we can move the modification to launch files out of scope.

Reply:
Updated to keep this line as same. Just changed the param file's name to make it suitable for `node_name.param.yaml `convention.

Reply:
Please review again @mitsudome-r